app/src/main/java/org/stepic/droid/ui/adapters/viewhoders/HeaderItemViewHolder.kt

```diff
@@ -0,0 +1,16 @@
+package org.stepic.droid.ui.adapters.viewhoders
+
+import android.view.View
+import org.stepic.droid.model.CoursesDescriptionContainer
+import org.stepic.droid.ui.custom.PlaceholderTextView
+import org.stepic.droid.util.ColorUtil
+
+class HeaderItemViewHolder(view: View, private val descriptionContainer: CoursesDescriptionContainer): CourseViewHolderBase(view) {
```

Comment:
It is good chance to refactor logic of extending `CourseViewHolderBase`. Let's remove it and extend `RecyclerView.ViewHolder`. So, we can cast holders in `onBindView` and execute methods without strange mutable properties. It is relevant for `FooterItemViewHolder` too.